server/views/pages/prisons/categoryGroups/addCategoryGroup.njk

```diff
@@ -0,0 +1,95 @@
+{% extends "partials/layout.njk" %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{%- from "moj/components/sub-navigation/macro.njk" import mojSubNavigation -%}
+{% from "moj/components/page-header-actions/macro.njk" import mojPageHeaderActions %}
+{%- from "govuk/components/button/macro.njk" import govukButton -%}
+{%- from "govuk/components/input/macro.njk" import govukInput -%}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+
+{% set pageHeaderTitle = 'Add a prisoner category group' %}
+{% set pageTitle = applicationName + " - Prisoner category groups - " + pageHeaderTitle %}
+
+{% set primaryNavItems = [
+  {
+    text: "Home",
+    href: "/"
+  },
+  {
+    text: "Prisons",
+    href: "/prisons",
+    active: true
+  }
+] %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+
+      <h1 class="govuk-heading-l">
+        <span class="govuk-caption-l">{{ prison.name }}</span>
+        {{ pageHeaderTitle }}
+      </h1>
+
+    </div>
+  </div>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+
+      {% include "partials/errorSummary.njk" %}
+
+      {# Build prisoner category array for checkboxes #}
+      {% set prisonerCategoryItems = [] %}
+      {% for prisonerCategory, description in prisonerCategories %}
+        {% set prisonerCategoryItems = (prisonerCategoryItems.push({
+          value: prisonerCategory,
+          text: description
+        }), prisonerCategoryItems)%}
+      {% endfor %}
+
+      <form action="/prisons/{{prison.code}}/category-groups/add" method="POST" novalidate>
+        <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+
+        {{ govukInput({
+          label: {
+            html: 'Prisoner category group name',
```

Comment:
```suggestion
            text: 'Category group name',
```
Leave off 'prisoner' for consistency and plaintext so `text` can be used instead of `html`